common/svc/beep.c

```diff
@@ -45,11 +45,7 @@ static uint16_t SECTION_INFOMEM beep_hour_freq = 2000;
 static uint8_t  SECTION_INFOMEM beep_hour_enable = 1;
 static uint16_t SECTION_INFOMEM beep_hour_duration = 10;
 static uint8_t  SECTION_INFOMEM beep_hour_quiet_enable = 0;
-static svc_beep_hour_quiet_t beep_hour_quiet[2] = {
-	{.h=21, .m=59},
-	{.h=06, .m=59}
-};
-static uint8_t beep_hour_muted = 0;
+static uint8_t  beep_hour_quiet[2] = {21, 06};
```

Comment:
Oh, I see. Just decrease the number of alarms to something more sensible like 10 or so.